Merge Rewordings to Main (#2)
arunaengine/presentations: src/k8s_mortality.md

1. [Introduction](#introduction)
2. [What Makes a Workload "Ultra-Long Running"?](#what-makes-a-workload-ultra-long-running)
3. [Understanding Pod Mortality: The Inevitable and the Preventable](#understanding-pod-mortality-the-inevitable-and-the-preventable)
   1. [Avoidable Failures: The Human Factor](#avoidable-failures-the-human-factor)
   2. [Unavoidable Failures: The Forces of Nature](#unavoidable-failures-the-forces-of-nature)
4. [Strategies for Success](#strategies-for-success)
   1. [Selecting the Right Tool](#selecting-the-right-tool)
   2. [Kubernetes Configuration Best Practices](#kubernetes-configuration-best-practices)
   3. [Example in Practice](#example-in-practice)
   4. [The Art of Pod Placement](#the-art-of-pod-placement)
   5. [A Real-World Example: Computational Biology Workloads](#a-real-world-example-computational-biology-workloads)
   6. [Container Necromancy: CRIU to the Rescue](#container-necromancy-criu-to-the-rescue)
5. [Summary and Outlook](#summary-and-outlook)

## Introduction

> ["Pods are mortal"](https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro/) - this frequently cited credo from the Kubernetes documentation has long been used to argue against running stateful workloads on Kubernetes. Pods are ephemeral, which is great for resilient microservices, but not so much for workloads that need to stick around and be stable over time.

Reviewer: ```suggestion
["Pods are mortal"](https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro/) - this frequently cited credo from the Kubernetes documentation has long been used to argue against running stateful workloads on Kubernetes. The ephemeral nature of pods, while perfect for resilient microservices, was supposedly unsuitable for workloads that required persistence and long-term stability.
```

#### Resource Efficiency and Predictability

When we're looking at workloads that extend over days or even weeks, it's really important to make sure we're being as efficient as we can with our resources. Memory management is especially important because RAM constraints and out-of-memory events often cause pod failures in Kubernetes. Tools that don't play by the rules in terms of memory usage or have sudden spikes in demand can cause big problems. Teams often have to overprovision resources as a precaution.  The best solution should show consistent resource use and support configurable internal memory limits that respect defined limits.

Think about how useful a tool is in practice. If it gets the job done in three days with a steady amount of resources, it's probably more valuable than one that promises to be faster but uses more resources in a random way.

> If you can change how long it takes to do a task, look for tools that let you switch between being precise and being fast. This way, you can find a balance that works for you. Shorter runtimes are good, but only if they use the same amount of resources and work well over time.

Reviewer: ```suggestion
If you can change how long it takes to do a task, look for tools that let you switch between being precise and being fast. This way, you can find a balance that works for you. Shorter runtimes are good, but only if they have a similar resource consumption profile and work well over time.
```
